public/components/custom_panels/panel_modules/panel_grid/panel_grid_so.tsx

```diff
@@ -153,17 +153,14 @@ export const PanelGridSO = (props: PanelGridProps) => {
   };
 
   // Save Visualization Layouts when not in edit mode anymore (after users saves the panel)
-  const saveVisualizationLayouts = useCallback(
-    async (panelID: string, visualizationParams: any) => {
-      const newVisualizations = updateLayout(panel.visualizations, visualizationParams);
-      const updateRes = await coreRefs.savedObjectsClient?.update('observability-panel', panelID, {
-        ...panel,
-        visualizations: newVisualizations,
-      });
-      setPanelVisualizations(updateRes?.attributes?.visualizations || []);
-    },
-    [panel]
-  );
+  const saveVisualizationLayouts = async (panelID: string, visualizationParams: any) => {
```

Comment:
what exactly is this change doing/what bug?

Reply:
* Saving after edit in SOPanels still has drag/drop control on visualizations
* Resize and drag visualization workflow post edit and save doesn't work in SOpanels, when a user hits the edit followed by save button, before response is made by the callback.